### DISCUSSION TITLE
API creation?

### DISCUSSION THREAD

**Discussion by sgaumin** [Synthoid/ExportSheetData#153] [Ideas]
After getting my hands on the **Google API Sheets** (which is working all fine) with **C# .NET**, I was thinking that it could be really helpful to have control over ESD extension as well. 

Please how does that sound? 

Thank you in advance! 
Loving this tool 👍

**Comment by Synthoid** [comment]
This sounds interesting. That said, I'm not sure if it's possible to control sheet add-ons with external API calls. I remember looking around for that kind of functionality and it just wasn't readily available. If it is possible, I can see how that would be useful for a lot of potential operations though.
